app/src/main/java/com/android/bookswap/ui/map/Map.kt

```diff
@@ -311,108 +308,104 @@ private fun DraggableMenu(listAllBooks: List<DataBook>) {
                   color = ColorVariable.BackGround,
                   shape =
                       RoundedCornerShape(
-                          topStart = DRAGGABLE_MENU_CORNER_RADIUS_DP.dp,
-                          topEnd = DRAGGABLE_MENU_CORNER_RADIUS_DP.dp))
+                          topStart = HEIGHT_RETRACTED_DRAGGABLE_MENU_DP.dp,
+                          topEnd = HEIGHT_RETRACTED_DRAGGABLE_MENU_DP.dp))
               .testTag("mapDraggableMenu")) {
-        Column(
-            modifier =
-                Modifier.padding(vertical = HANDLE_HEIGHT_DP.dp)
-                    .fillMaxWidth()
-                    .testTag("mapDraggableMenuStructure")) {
-              // draggable handle
-              Box(
+        Column(modifier = Modifier.fillMaxWidth().testTag("mapDraggableMenuStructure")) {
+          // draggable handle
+          Spacer(modifier = Modifier.height(HANDLE_HEIGHT_DP.dp))
+          Box(
+              modifier =
+                  Modifier.align(Alignment.CenterHorizontally)
+                      .width(HANDLE_WIDTH_DP.dp)
+                      .height(HANDLE_HEIGHT_DP.dp)
+                      .background(
+                          color = ColorVariable.AccentSecondary,
+                          shape = RoundedCornerShape(HANDLE_CORNER_RADIUS_DP.dp))
+                      .testTag("mapDraggableMenuHandle"))
+          Spacer(modifier = Modifier.height(SPACER_HEIGHT_DP.dp))
+          HorizontalDivider(
+              modifier = Modifier.fillMaxWidth().testTag("mapDraggableMenuHandleDivider"),
+              thickness = DIVIDER_THICKNESS_DP.dp,
+              color = ColorVariable.Accent)
+          LazyColumn(userScrollEnabled = true, modifier = Modifier.fillMaxHeight()) {
+            itemsIndexed(listAllBooks) { index, book ->
+              Spacer(modifier = Modifier.height(PADDING_VERTICAL_DP.dp))
+              Row(
```

Comment:
You should probably add an onClick(), it might be empty right now, but we will need it at some point, it will redirect to it's profile normally.